mne/source_estimate.py

```diff
@@ -1352,10 +1278,9 @@ def center_of_mass(self, subject=None, hemi=None, restrict_vertices=False,
 
     def plot(self, subject=None, surface='inflated', hemi='lh',
              colormap='auto', time_label='time=%0.2f ms',
-             smoothing_steps=10, fmin=None, fmid=None, fmax=None,
-             transparent=None, alpha=1.0, time_viewer=False,
-             config_opts={}, subjects_dir=None, figure=None,
-             views='lat', colorbar=True, clim=None):
+             smoothing_steps=10, transparent=None, alpha=1.0,
+             time_viewer=False, config_opts={}, subjects_dir=None, figure=None,
```

Comment:
@Eric89GXL this is opens the door to state-relate bug when the function is called multiple times, the the dict will still be the same. A no-go. We should put it to None and handle it inside.